internal/command/clusters/uninstall.go

```diff
@@ -0,0 +1,218 @@
+package clusters
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	certmanagerv1 "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
Nit: `cmapi` (this is a convention we use pretty much everywhere i.e https://github.com/cert-manager/approver-policy/blob/main/pkg/approver/manager/manager.go#L22

Reply:
Addressed in 71eab72d09063f2ef5c7251948d38d7cee9a658b